central/node/datastore/internal/store/common/v2/merge.go

```diff
@@ -48,6 +50,18 @@ func mergeComponents(parts *NodeParts, node *storage.Node) {
 		// Generate an embedded component for the edge and non-embedded version.
 		node.Scan.Components = append(node.Scan.Components, generateEmbeddedComponent(cp))
 	}
+
+	sort.SliceStable(node.GetScan().GetComponents(), func(i, j int) bool {
```

Comment:
for readability can we set a temp var?

components := node.GetScan().GetComponents()